sql/src/main/java/io/crate/executor/QueryResult.java

```diff
@@ -28,6 +29,16 @@
 
 public class QueryResult implements TaskResult {
 
+    public static final Function<Bucket, TaskResult> TO_TASK_RESULT = new Function<Bucket, TaskResult>() {
```

Comment:
is this the best place for this? It's returning TaskResult but is in the QueryResult class?